nix/sources.json

```diff
@@ -0,0 +1,14 @@
+{
+  "launch-deversifi": {
+    "branch": "dev",
+    "description": "Instructions to deploy the required services for DeversiFi Exchange",
+    "homepage": "",
+    "owner": "rhinofi",
+    "repo": "launch-deversifi",
+    "rev": "2fe7a3fd74c4d6afc09249741fd2be97598fb48e",
+    "sha256": "1r5zrvp6fvk2rx09hqxh4h9fjmibzkfiv7r5pfhm636lb43zwc0f",
+    "type": "tarball",
+    "url": "https://github.com/rhinofi/launch-deversifi/archive/2fe7a3fd74c4d6afc09249741fd2be97598fb48e.tar.gz",
+    "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
```

Comment:
logger is in a public repo, can we use our public `nixpkgs` here instead?

Reply:
No, because we need the dprint and eslint wrappers.